apache/superset
# feat: show build number value in the About if present in the config

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
![aboutafter](https://user-images.githubusercontent.com/62111795/120540066-412f1180-c3b6-11eb-9489-7eed487e908f.jpg)

### TESTING INSTRUCTIONS
Add a variable called BUILD_NUMBER in the config.py, the value should be displayed in the About section underneath the version/sha. No build information is shown if there is no variable BUILD_NUMBER in the config.py file.

### ADDITIONAL INFORMATION
